src/config-loader.ts

```diff
@@ -164,22 +163,21 @@ async function getConfigFilesUp() {
 /**
  * Gets all svelte config from a specified `lookupDir`.
  *
- * @param {string} lookupDir
+ * @param lookupDir
  * Directory in which to look for svelte files.
  *
- * @returns {Promise<string[]>}
+ * @returns
  * Returns an array containing all available config files.
  */
-async function getConfigFiles(lookupDir) {
-  /** @type {[string, boolean][]} */
-  const fileChecks = await Promise.all(
+async function getConfigFiles(lookupDir: string): Promise<string[]> {
```

Comment:
We can convert this whole function to be sync now, right?

Reply:
Correct, no more promises.